js/renovation/button.tsx

```diff
@@ -5,12 +5,14 @@ import Widget from './widget';
 
 const WidgetJSX = JSXConstructor<Widget>(Widget);
 
-const getImageContainerJSX = (source: string) => {
+const getImageContainerJSX = (source: string, position: string = '') => {
```

Comment:
You have already defined the `position` property in the definition.

BTW, we should avoid defining defaults to function arguments.